src/hooks-worker-client.coffee

```diff
@@ -8,9 +8,6 @@ generateUuid = require('node-uuid').v4
 logger = require './logger'
 which = require './which'
 
-
-HOOK_TIMEOUT = 5000
```

Comment:
I think we can get rid of these constants at all and move the defaults to the constructor and override them if the configuration is present.